docs/se/guides/07_db_client.adoc

```diff
@@ -0,0 +1,634 @@
+import io.helidon.dbclient.DbClient;
+
+...
+
+Config config = Config.create(); // Landmark to add DB client
+
+DbClient dbClient = DbClient.builder()
+                .config(config.get("db"))       // <2>
+                .addService(DbClientMetrics.counter().statementNames("select-book"))   // <3>
+                .build();
+----
+<1> Add import statements
+<2> Configure the DB Client with the "db" section of application.yaml.
+<3> Add a counter for metrics.
+
+The DB Client metric counter will be executed only for `select-book` statement, and check how many times it was invoked.
+At this point, the database is empty, and needs to be initialised. To achieve that, the DB Client can be used
+to create a table in the database.
+
+[source,java]
+.Insert a createTable method below the dbClient:
+----
+DbClient dbClient = DbClient.builder()
+                .config(config.get("db"))
+                .addService(DbClientMetrics.counter().statementNames("select-book"))
+                .build();
+
+createTable(dbClient);
+----
+
+[source,java]
+.Use the DbClient to build a table:
+----
+private static void createTable(DbClient dbClient) {
+        dbClient.execute(exec -> exec.namedDml("create-table"))     // <1>
+                .await();
+    }
+----
+<1> Use the "create-table" script to build a table with book name and information.
+
+The `createTable` is invoked only once and create an empty table with two columns: name and info. It is used to boostrap
+the server, so the `await` method is called in this particular case because the table must be created before the server
+starts. A new service can manage request to interact with this table which represent our library.The services are
+registered in the `createRouting` method.
+
+[source,java]
+.Modify the createRouting method:
+----
+import io.helidon.dbclient.health.DbClientHealthCheck;
+
+...
+
+WebServer server = WebServer.builder(createRouting(config, dbClient))   // <1>
+                .config(config.get("server"))
+                .addMediaSupport(JsonpSupport.create())
+                .build();
+
+...
+
+private static Routing createRouting(Config config, DbClient dbClient) {
+        HealthSupport health = HealthSupport.builder()
+                .addLiveness(DbClientHealthCheck.create(dbClient))  // <2>
+                .build();
+
+        return Routing.builder()
+                .register(health)                   // Health at "/health"
+                .register(MetricsSupport.create())                  // Metrics at "/metrics"
+                .register("/greet", new GreetService(config))
+                .register("/library", new LibraryService(dbClient))  // <3>
+                .build();
+    }
+----
+<1> Add dbClient as a parameter of createRouting method.
+<2> Add Health check to control the application behavior.
+<3> Register the LibraryService to the Routing.
+
+The library service does not exist yet but that is the next step of the guide. It has a  constructor with the
```

Comment:
The library service does not exist yet but that is the next step of the guide. It has a constructor with the

[remove the extra space before constructor]